src/main/java/org/mifos/phee/billPay/PheeBillPayApplication.java

```diff
@@ -1,14 +1,14 @@
-package org.mifos.pheebillpay;
```

Comment:
if you can send me this rename alone as a new PR, it would be easy to review and merge. Coz this PR looks like a code dump